docs/guides/insar_product_guide.md

```diff
@@ -0,0 +1,284 @@
+| ms to sec | along-track | ocean currents, moving object detection, MTI | 
+| days | differential | glacier/ice fields/lava flows, surface water extent, hydrology |
+| days to years | differential | subsidence, seismic events, volcanic activity, crustal displacement | 
+
+*Table 1: Temporal baselines and what they measure. Different geophysical phenomena can be detected based upon the temporal baseline. In general, the longer the temporal baseline, the smaller the motion that can be detected.*
+
+## InSAR Workflow
+
+The InSAR workflow used in HyP3 was developed by ASF using GAMMA software.  The steps include pre-processing steps, interferogram preparation, and product creation.  Once these steps are performed, an output product package will be created.  See [product packaging](#product-packaging) for details on the individual files in the package.  
+
+### Pre-Processing
+
+Pre-processing steps prepare the SAR images to be used in interferometry.  The pre-processing steps include image selection, ingest (including calibration), creation of a suitable DEM, and calculation of the burst overlap.
+
+#### Finding an InSAR Pair
+
+Although it is possible to start from RAW data, Sentinel-1 InSAR processing is typically done with [Interferometric Wide swath Single Look Complex](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/acquisition-modes/interferometric-wide-swath "Link to ESA IW SLC description") (IW SLC) data.  This means that the data has been formed into an image through SAR processing, but has not been multi-looked.  When selecting an InSAR pair, observe the following required conditions:
+
+1. Images from an identical orbit direction (either ascending or descending)
+2. Images with identical incidence angles and beam mode
+3. Images with identical resolution and wavelength (usually from the same sensor)
+4. Images with the same viewing geometry (same track and frame)
+5. Images with identical polarizations (both HH or VV)
+
+In addition, the following suggestions may be helpful:
+
+1. Use images from similar seasons/growth/weather conditions
+2. For deformation mapping: limited spatial separation of acquisition locations (small physical baseline)
+3. For topographic mapping: limited time separation between images (small temporal baseline) 
+
+------
+> **Aside: Critical Baseline**
+
+> Large baselines are better than small for deformation detection. However, as the baseline increases, coherence decreases. As some point, it is impossible create an interferogram because of baseline decorrelation. The maximum viable baseline per platform, referred to as the *critical baseline*, is a function of the distance to the ground, the wavelength, and the viewing geometry of the platform. For Sentinel-1 this critical baseline is about 5 KM. In practice if the perpendicular baseline between images is more than 3/4 of the critical baseline, interferogram creation will be problematic due to the level of noise. 
```

Comment:
I thought larger baselines were only beneficial for topographic applications, and that the shorter the better for deformation mapping.

Reply:
You are correct, should read 

"Large baselines are better than small for topographic measurements."

Reply:
Resolved by PR #95